misk-sampling/build.gradle.kts

```diff
@@ -9,6 +9,7 @@ plugins {
 dependencies {
   api(libs.guava)
   api(libs.jakartaInject)
+  api(libs.micrometerCore)
```

Comment:
I think you probably want Misk metrics? The Micrometer migration wasn't completed.